zkevm-circuits/src/copy_circuit.rs

```diff
@@ -131,6 +131,18 @@ impl<F: Field> SubCircuitConfig<F> for CopyCircuitConfig<F> {
         let rwc_inc_left = copy_table.rwc_inc_left;
         let tag = copy_table.tag;
 
+        // annotate table columns
+        tx_table.annotate_columns(meta);
+        rw_table.annotate_columns(meta);
+        bytecode_table.annotate_columns(meta);
+        copy_table.annotate_columns(meta);
+
+        // annotate circuit specific columns
+        meta.annotate_lookup_any_column(is_last, || "is_last");
+        meta.annotate_lookup_any_column(value, || "value");
+        meta.annotate_lookup_any_column(is_code, || "is_code");
+        meta.annotate_lookup_any_column(is_pad, || "is_pad");
```

Comment:
@adria0 are all these, like is_pad lookup columns? If not perhaps they have to be annotated with `region.name_column`?

Reply:
To annotate columns that are used as lookup and don't have a `TableColumn` type. NOTE that within regions, these will be overwritten by the region annotations unless the error is a lookup one.

Reply:
so I see `is_pad` that is used in many gates and as part of the condition of lookups, but not like something that has to match in the lookup. Should we annotate `is_pad` the other way? In the synthesise with the region?

Reply:
> > To summarize, I agree with @leolara . The columns that are specific to the circuit and aren't used as lookup, should be annotated inside the `Region`.
> 
> IMO we should add this comment to the method documentation

Feel free to file an issue in our fork!! :)